scripts/local_k8s/unittests/test_properties.py

```
@@ -0,0 +1,232 @@
+        commands.append((command, args))
+        return object()
+
+    config = _preview_config(release_name="wildside", chart_name="wildside")
+    monkeypatch.setattr("local_k8s.cluster.require_tools", lambda _: None)
+    monkeypatch.setattr("local_k8s.cluster.run", record_run)
+    monkeypatch.setattr("local_k8s.cluster._remove_stale_archive", lambda _: None)
+    # This property exercises only the archived image name, so stub the archive
+    # path helper to avoid touching the filesystem for each generated example.
+    monkeypatch.setattr(
+        "local_k8s.cluster._image_archive_path",
+        lambda _config, *, archive_dir=None: Path(archive_dir or "/tmp")
```

Ruff (0.15.21)

[error] Probable insecure usage of temporary file or directory: "/tmp"

(S108)